src/packages/solid/components/Wave/Wave.tsx

```diff
@@ -0,0 +1,131 @@
+import { View } from '@lightningjs/solid';
+import type { UIComponentProps } from '../../types/interfaces.js';
+import styles from './Wave.styles.js';
+
+export interface WaveProps extends UIComponentProps {}
+
+const Wave: Component<WaveProps> = (props: WaveProps) => {
+
+  let right, left, center, rightCenter, leftCenter, Container;
+  
+  // @ts-expect-error
+  const maxHeight = createMemo(() => props.height ?? styles.Container.tones[props.tone ?? styles.tone].height ?? styles.Container.base.height);
+
+  
+  const isAnimating = createMemo(() => props.toggleAnimation);
+  
+  function startAnimation() {
+
+    if (isAnimating()) {
+    // right animation
+    left.chain({ height: (maxHeight()/3)}, { duration: 300})
+    .chain({ height: (maxHeight()/3)}, { duration: 50})
+    .chain({ height: (maxHeight()/2)}, { duration: 300})
+    .chain({ height: (maxHeight()/2)}, { duration: 50})
+    .chain({ height: (maxHeight())}, { duration: 300})
+    .chain({ height: maxHeight()/4}, { duration: 300})
+    .chain({ height: maxHeight()/4}, { duration: 50}).start();
+
+    // center left animation 2-
+    leftCenter.chain({ height: (maxHeight()/2)}, { duration: 300})
+    .chain({ height: (maxHeight()/2)}, { duration: 50})
+    .chain({ height: (maxHeight()/3)}, { duration: 300})
+    .chain({ height: (maxHeight()/3)}, { duration: 50})
+    .chain({ height: (maxHeight()/4)}, { duration: 300})
+    .chain({ height: maxHeight()}, { duration: 300})
+    .chain({ height: maxHeight()}, { duration: 50}).start();
+
+    // // center animation
+    center.chain({ height: maxHeight()}, { duration: 300 })
+    .chain({ height: maxHeight()}, { duration: 50 })
+    .chain({ height: (maxHeight()/4)}, { duration: 300 })
+    .chain({ height: (maxHeight()/4)}, { duration: 50 })
+    .chain({ height: maxHeight()}, { duration: 300 })
+    .chain({ height: (maxHeight())}, { duration: 50 })
+    .chain({ height: (maxHeight()/4)}, { duration: 300 }).start();
+
+    // // center right animation
+    rightCenter.chain({ height: (maxHeight()/2)}, { duration: 300})
+    .chain({ height: (maxHeight()/2)}, { duration: 50})
+    .chain({ height: (maxHeight()/3)}, { duration: 300})
+    .chain({ height: (maxHeight()/3)}, { duration: 50})
+    .chain({ height: (maxHeight()/4)}, { duration: 300})
+    .chain({ height: maxHeight()}, { duration: 300})
+    .chain({ height: maxHeight()}, { duration: 50}).start();
+
+    // right animation
+    right.chain({ height: (maxHeight()/3)}, { duration: 300})
+    .chain({ height: (maxHeight()/3)}, { duration: 50})
+    .chain({ height: (maxHeight()/2)}, { duration: 300})
+    .chain({ height: (maxHeight()/2)}, { duration: 50})
+    .chain({ height: (maxHeight())}, { duration: 300})
+    .chain({ height: maxHeight()/4}, { duration: 300})
+    .chain({ height: maxHeight()/4}, { duration: 50}).start();
+   }
+  }
+
+  createEffect(() => {
+    setInterval(() =>
+      startAnimation(), 1450);
+    }, isAnimating());
```

Comment:
two things
 - is the 1450 completely just trial and error? 
 - what's isAnimating() doing here? mostly im just less familiar with the createEffect API

Reply:
the sum of the durations is 1350, 1450 adds some buffer room so it doesn't start before the animation is completely finished.

isAnimating() is actually not needed here..oops